commercedbsync/src/com/sap/cx/boosters/commercedbsync/repository/platform/MigrationHybrisMSSqlPlatform.java

```diff
@@ -10,27 +10,32 @@
 import de.hybris.bootstrap.ddl.HybrisPlatform;
 import de.hybris.bootstrap.ddl.sql.HybrisMSSqlBuilder;
 import org.apache.ddlutils.DatabaseOperationException;
+import org.apache.ddlutils.DdlUtilsException;
 import org.apache.ddlutils.Platform;
 import org.apache.ddlutils.PlatformInfo;
 import org.apache.ddlutils.model.Column;
 import org.apache.ddlutils.model.Database;
+import org.apache.ddlutils.model.Index;
 import org.apache.ddlutils.model.Table;
 import org.apache.ddlutils.platform.DatabaseMetaDataWrapper;
+import org.apache.ddlutils.platform.JdbcModelReader;
 import org.apache.ddlutils.platform.mssql.MSSqlModelReader;
 import org.apache.ddlutils.platform.mssql.MSSqlPlatform;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.sql.Connection;
-import java.sql.SQLException;
-import java.sql.Types;
+import java.sql.*;
+import java.util.Collection;
+import java.util.Iterator;
 import java.util.Map;
 import java.util.Set;
 
 public class MigrationHybrisMSSqlPlatform extends MSSqlPlatform implements HybrisPlatform {
 
     private static final Logger LOG = LoggerFactory.getLogger(MigrationHybrisMSSqlPlatform.class);
 
+    private static final String[] KNOWN_SYSTEM_TABLES = new String[]{"dtproperties"};
```

Comment:
DB Sync is designed to handle only SAP Commerce type system tables, custom tables should be handled by `migration.data.tables.custom` property. Any particular reason for such customization to be hardcoded here?

Reply:
If this is related with Database Diagrams used in older versions of MSSQL, there is no intent to handle such in core functionality of DB Sync